pages/index.vue

```diff
@@ -153,12 +253,102 @@
         window.open(`${STORAGE_URL}/${file.name}`);
         return false;
       },
+      /**
+       * Download archive
+       */
+      download(item) {
+        window.open(`${STORAGE_URL}/${item.name}`);
+        return false;
+      },
     },
   };
 </script>
 
-<style>
+<style lang="scss">
+  $primary: #251B5B;
+
+  .container {
+    padding: 24px 0;
+  }
+
   .files {
     padding-top: 15px;
   }
+
+  .theme--light.v-table,
+  table.v-table {
+    thead {
+      th {
+        background-color: $primary;
+      }
+    }
+    tbody {
+      tr {
+        &:not(:last-child) {
+          border: none;
+        }
+        &.error {
+          background-color: rgba(244, 67, 54, .1) !important;
```

Comment:
I guess this should work even without the `!important` mark. Please check and remove it if possible. Also true for the other occurrences.

Reply:
No it won't work without the !important because of material js :/